### ISSUE TITLE
Retweets of a tweet are grouped, but still duplicated

### ISSUE THREAD

**Issue by punchagan** [baali/SoFee#7]
![tue nov 29 21 00 45 ist 2016](https://cloud.githubusercontent.com/assets/315678/20716044/1d218026-b677-11e6-92ba-41027c57defd.jpg)

**Comment by baali** [comment]
Yes, I will use the 'distinct' in my django query for links to fix this one.
